Sources/HummingbirdElementary/HTMLResponse.swift

```diff
@@ -25,6 +25,9 @@ public struct HTMLResponse<Content: HTML & Sendable>: Sendable {
     /// The default is 1024 bytes.
     public var chunkSize: Int
 
+    /// Response headers
```

Comment:
could you add a bit more docc comment to explain that this can be used to add additional custom headers to a response, maybe even with a snippet?

Reply:
I've added docc, let me know if it is enough